### DISCUSSION TITLE
Support to reset consumed energy

### DISCUSSION THREAD

**Discussion by kamkilt** [Koenkk/zigbee2mqtt#6939] [Devices]
Is there any way to reset sum of consumed energy on smart sockets?

I use Blitzwolf BW-SHP13, but as far I searched, there isn't any option to reset energy consumption on any socket that counts consumed energy.

I can reset counted energy on my z-wave and esp smart sockets, on zigbee sockets I can't.

Is this limitation of zigbee2mqtt or available smart sockets?

**Comment by Calimerorulez** [comment]
+1, same question here...

**Reply by playersct** [reply]
I think it's limitation smart sockets

**Reply by sjorge** [reply]
I was looking at the ZCL a while back for this without luck, so far the only way I found was to reset the device

**Comment by MrDrache333** [comment]
@playersct @sjorge @kamkilt 
Now there is a possibility https://www.zigbee2mqtt.io/devices/TS011F_plug_1.html#reset-energy

Ive tested it and it works, but I had to re-pair the socket. Would be nice to automate this using the JSON-Payload but the re-paring is a bit od.
